pom.xml

```diff
@@ -4,8 +4,9 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <groupId>com.microsoft.office</groupId>
+    <groupId>com.microsoft.ewsjavaapi</groupId>
     <artifactId>ews-java-api</artifactId>
+    
     <version>1.3-SNAPSHOT</version>
```

Comment:
hmm never heard of a tool which would give us that information. Maybe this is ~~"just"~~ about looking through the git-log? But I appreciate the idea of deploying `2.0-Snapshot` releases now since we had some greater changes and there will possibly join in more of these until we are releasing the first stable one.